app/controllers/api/v1/accounts/whatsapp/reauthorizations_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::Accounts::Whatsapp::ReauthorizationsController < Api::V1::Accounts::BaseController
```

Comment:
quick note : this could be a part of the authorization controller as an update action ?

Reply:
bump